bin/docserv-stitch

```diff
@@ -110,30 +112,57 @@ for i in "$@"
       ;;
       --simplify)
         simplify_config=1
+        shift
       ;;
       --revalidate-only)
         revalidate=1
+        shift
       ;;
       --spinner)
         enable_spinner=1
+        shift
       ;;
       --valid-languages=*)
         valid_languages=" ${i#*=}"
+        shift
+      ;;
+      --site-sections=*)
+        site_sections="${i#*=}"
+        shift
       ;;
-      *)
-        if [[ -d $(readlink -f $i) ]] && [[ ! $input_dir ]]; then
-          input_dir=$(readlink -f $i)
-        elif [[ -d $(readlink -f $(dirname $i)) ]] && [[ ! $output_file ]]; then
-          output_file=$(readlink -f $i)
-        else
-          unknown+="  $i\n"
-        fi
+      --*)
+        out "Unknown option $i"
```

Comment:
This will exit after the first unknown parameter. That's something I tried to avoid previously, because if there are multiple wrong parameters, users will likely need multiple failed runs before all their parameter mistakes are fixed, each generating a different error message. That is a bad user experience.

Reply:
If we agree to implement the above structure, this will address your concerns about bad user experience.